cli/cmd/network_report.go

```diff
@@ -54,7 +47,6 @@ replicated network report <network-id> --watch`,
 
 	cmd.Flags().BoolVarP(&r.args.networkReportWatch, "watch", "w", false, "Watch for new network events in real-time (polls every 2 seconds)")
 	cmd.Flags().BoolVar(&r.args.networkReportSummary, "summary", false, "Get aggregated report summary with statistics instead of individual events")
-	cmd.Flags().BoolVar(&r.args.networkReportShowExternalOnly, "show-external-only", true, "Show only external network traffic")
```

Comment:
i wonder if we want to leave this flag added but hidden, so if vendors have this in their CI scripts it won't fail. optionally we can display a deprecated warning as well if the hidden flag is supplied